src/code_0804B9F8.c

```diff
@@ -0,0 +1,54 @@
+#include "global.h"
+#include "utils.h"
+#include "fileScreen.h"
+
+
+
+#define gPaletteRam ((u8*)(0x5000000))
+
+extern u16 gPaletteBuffer[];
+extern u32 gUsedPalettes;
+
+// function pointer to overlay (0x03005e98) in ram calls rom function MakeFadeBuff256
+extern u32 gMakeFadeBuff256;
+typedef void (*fptrMakeFadeBuff256)(u8*,u8*,u16,u8);
+
+
+void FadeMain()
+{
```

Comment:
Clang format please

Reply:
sorry how do i make sure this gets run on every file because i screwed it up the first time whoops.

Reply:
If you have clang-format installed, you should be able to run the `format.sh` in the root folder of this repo.